src/components/browse/components/FacetList/FacetOfFacets.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Collapse } from './../../../ui/Collapse';
+import { Fade } from './../../../ui/Fade';
+
+/**
+ * Used to render individual facet fields and their available terms in FacetList.
+ *
+ * @memberof module:facetlist
+ * @class Facet
+ * @type {Component}
```

Comment:
(Not Important Thoughts)
Is not really important to put too much stuff into docstring, lately have only been trying to add in docstrings for info that can't easily intuited from the code (e.g. usually not `@class`) but just to cut down on amount of lines of code sometimes.

At one point we had tried to use JSDoc, extract docstring, build reference documentation, and put it up on ReadTheDocs (docs website for Python libraries) which presented more work than was really worth. JSDoc required some verbose docstring, when working with ES5+ (classes, etc) especially. We then went to ESDoc, which was nicer, partly because it could intuit 'type', 'class' and other things which JSDoc couldn't without explicit docstrings. Then kind of stopped since it seems more useful to just have VS Code parse doc strings and give them when hover over an imported function/component or similar.

Reply:
If interested, the probably 2y+ old reference (JS stuff based on JSDoc) still persists at 4dn-dcic.github.io/fourfront/.
As can tell the JS docs aren't too useful.
We are able to build much nicer "ESDoc" reference, but were some (CSS/styling?) glitches when tried to upload to doc.esdoc.org and haven't had much time to look into further.

Reply:
I swear there's new pieces of 4DN and CGAP related documentation that I discover almost every day. Thanks!